Services/PredefinedValuesProductAttributeService.cs

```diff
@@ -39,6 +39,7 @@ select accseq.Concat(new[] { item })
         public IEnumerable<ProductAttributeDescription> GetProductAttributesRestrictedToPredefinedValues(ContentItem product)
             => _productAttributeService
                 .GetProductAttributeFields(product)
-                .Where(x => x.Settings is IPredefinedValuesProductAttributeFieldSettings textSettings && textSettings.RestrictToPredefinedValues);
+                .Where(x => x.Settings is IPredefinedValuesProductAttributeFieldSettings textSettings && textSettings.RestrictToPredefinedValues)
+                .OrderBy(x => $"{x.PartName}.{x.Name}");
```

Comment:
Don't format strings just to do a double sort order. Do `OrderBy(x => x.PartName).ThenBy(x => x.Name)` instead.

Reply:
Fixed